### PR TITLE
Remove cache_control for random - Bug: T269569

### PR DESCRIPTION
**Pull request by ArtTsymbar**
Remove cache_control for random

Bug: T269569

### CODE REVIEW

**Comment by thesocialdev** [comment]
@clarakosi and @Pchelolo can you take a look at the code?

File: projects/v1/default.wmf.yaml [hunk 1]
```diff
@@ -67,7 +67,7 @@ paths:
       - path: v1/related.js
         options: '{{options.related}}'
       - path: v1/random.yaml
-        options: '{{merge({"random_cache_control": "s-maxage=2, max-age=1"}, options.wikifeeds)}}'
+        options: '{{options.wikifeeds)}}'
```
Comment (clarakosi) [inline projects/v1/default.wmf.yaml:70]: extra `)` at the end there. Same with the others below

Comment (ArtTsymbar) [reply projects/v1/default.wmf.yaml]: Hi, @clarakosi , thanks! ")" removed.

**Comment by Pchelolo** [comment]
needs a manual rebase, after that can ve merged.

**Comment by ArtTsymbar** [comment]
> needs a manual rebase, after that can ve merged.

Separate PR created https://github.com/wikimedia/restbase/pull/1291 with the same changes. Thanks!

**Comment by Pchelolo** [comment]
> Separate PR created #1291 with the same changes. Thanks!

https://docs.github.com/en/github/getting-started-with-github/about-git-rebase

For future reference, here is what you should have done
git remote add wikimedia git@github.com:wikimedia/restbase.git
git remote add ArtTsymbar git@github.com:ArtTsymbar/restbase.git
git checkout master
git pull wikimedia master 
git checkout T269569-remove-cache-control-for-random
git rebase master
<resolve conflicts>
git push -f ArtTsymbar T269569-remove-cache-control-for-random

**Pull request closed without merging**